pages/common/cowsay.md

```diff
@@ -1,24 +1,28 @@
 # cowsay
 
-> Generate an ASCII character (by default a cow) saying or thinking something.
+> Print an ASCII friend (by default a cow) saying or thinking something.
 > More information: <https://github.com/tnalpgge/rank-amateur-cowsay>.
 
-- Print an ASCII cow saying "Hello world":
+- Print an ASCII cow saying "Hello World":
 
-`cowsay "Hello world"`
+`cowsay "{{Hello World}}"`
 
 - Read text from stdin for the balloon:
 
-`echo "Hello" | cowsay`
+`echo "{{Hello World}}" | cowsay`
 
-- List all available characters:
+- List all available friends:
 
 `cowsay -l`
 
-- Print an ASCII dragon saying "Hello":
+- Print an ASCII friend (different from default cow) saying "Hello World":
```

Comment:
```suggestion
- Print a custom ASCII friend saying "Hello World":
```
I appreciate the correct preposition ("from") after "different"! 😉

Reply:
I am unsure about this one, I feel since this is a fun "game" program, it should be a bit lighter and less serious than that.

Reply:
To me, they both seem equally (un)serious, and the latter is a bit easier to read 🤔